src/k2/cpo/service/CPOService.h

```diff
@@ -88,6 +90,12 @@ class CPOService {
 
     seastar::future<std::tuple<Status, dto::GetSchemasResponse>>
     handleSchemasGet(dto::GetSchemasRequest&& request);
+
+    seastar::future<std::tuple<Status, dto::MetadataPersistResponse>>
+    handleMetadataPersist(dto::MetadataPersistRequest&& request);
```

Comment:
I would rename this method to handleMetadataPut as it matches better its counterpart handleMetadataGet

Reply:
fixed